### DISCUSSION TITLE
Maximale Länge in Discussions

### DISCUSSION THREAD

**Discussion by IvanDeSchreck** [wulfland/BootCampBasta23#6] [Q&A] [answered]
Was ist die maximale Zeichenanzahl bei Eingaben in [Discussions](https://github.com/wulfland/BootCampBasta23/discussions) @wulfland  ❓

**Comment by wulfland** [comment]
Gute Frage 😏 Muss ich auch schauen.

**Comment by wulfland** [comment, marked as answer]
Max length: 65536 characters

**Comment by wulfland** [comment]
Siehe https://github.com/dead-claudia/github-limits

**Comment by paulchen8** [comment]
Danke für die Infos
